js/events/pointer/base.js

```diff
@@ -26,14 +26,20 @@ const BaseStrategy = Class.inherit({
     _handler: function(e) {
         const delegateTarget = this._getDelegateTarget(e);
 
-        return this._fireEvent({
+        const event = {
             type: this._eventName,
             pointerType: e.pointerType || eventSource(e),
             originalEvent: e,
             delegateTarget: delegateTarget,
             // NOTE: TimeStamp normalization (FF bug #238041) (T277118)
             timeStamp: browser.mozilla ? (new Date()).getTime() : e.timeStamp
-        });
+        };
+
+        if(e.originalEvent) {
+            event.target = e.originalEvent.target.shadowRoot ? e.originalEvent.path[0] : e.originalEvent.target;
```

Comment:
seems like we have excess change here: when there is no shadowRoot `event.target = e.originalEvent.target`
Shouldn't we write it this way:
```javascript
const originalEvent = e.originalEvent;
if(originalEvent?.target.shadowRoot) {
    event.target = e.originalEvent.path[0];
}
```

Reply:
ok